packages/calypso-e2e/src/lib/components/editor-toolbar-component.ts

```diff
@@ -446,7 +446,10 @@ export class EditorToolbarComponent {
 
 		// To support i18n tests.
 		const translatedTargetName = await this.translateFromPage( 'Options' );
-		const button = editorParent.getByRole( 'button', {
+		// Narrowing down to the "Editor top bar" is needed because it might conflict with
+		// the options button for the block toolbar, causing a strict-mode violation error
+		// due to duplicate elements on the page.
+		const button = editorParent.getByLabel( 'Editor top bar' ).getByRole( 'button', {
```

Comment:
I had to add the `getByLabel` here because of the following error:

```
    locator.getAttribute: Error: strict mode violation: locator('body.block-editor-page').getByRole('button', { name: 'Options', exact: true }) resolved to 2 elements:                                             
        1) <button type="button" aria-haspopup="true" aria-label="O…>…</button> aka getByLabel('Editor top bar').getByLabel('Options')                                                                              
        2) <button id=":r28:" type="button" tabindex="-1" data-comm…>…</button> aka getByLabel('Block tools').getByLabel('Options')        
```